### PR TITLE
added labelClasses support

### LINKED ISSUE #333: How to add label icons?
For example I'd like to have a fontawesome icon as label. I could paste this as label icon that I could then style with CSS:

    <i class="fa fa-envelope-o" aria-hidden="true"></i>

But currently it is parsed as text so can't use HTML?

### PR DESCRIPTION
**Pull request by zoul0813**
fixes #333 - adds "labelClasses" to the field schema, allowing you to configure icons and other styles to labels via CSS classes.

### CODE REVIEW

**Comment by zoul0813** [comment]
Not too familiar with your testing policies ... does the formGenerator test cover it?

File: src/formGenerator.vue [hunk 1]
```diff
@@ -571,3 +569,11 @@ div.vue-form-generator(v-if='schema != null')
 
 	} // fieldset
 </style>
+ // .errors
```
Comment (zoul0813) [inline src/formGenerator.vue]: oops ... how'd that get there?

**Comment by PushpinderRattan** [comment]
labelClasses field does not seem to be working for me as I am trying to add a fontawesome icon next to the field. Is there anyway to do so? Do you have any example?
Thank you.

**Comment by PushpinderRattan** [comment]
Please ignore it.

**Pull request merged**